Sources/Appwrite/WebSockets/HTTPHandler.swift

```diff
@@ -48,8 +48,6 @@ extension HTTPHandler : ChannelInboundHandler, RemovableChannelHandler {
         var headers = HTTPHeaders()
         
         headers.add(name: "Host", value: "\(client.host):\(client.port)")
-        headers.add(name: "Content-Type", value: "text/plain")
```

Comment:
is this intended and no longer required?

Reply:
Intended, this broke with proxies because some don't like content-type and content-length or a body when making a GET request